content/dstu2/contract.md

```diff
@@ -42,11 +42,11 @@ Notes:
 ### Response
 
 Search for all Contracts subject to a specific patient
-<%= headers 200, GET: '[...]/Contract?subject:Patient=14067775' %>
+<%= headers 200, GET: '[...]/Contract?subject:Patient=4598008' %>
```

Comment:
We usually use valid ids and urls so that someone could call them and see actual data. However, in the examples, we just make sure to adjust to obviously fake people to ensure someone doesn't Google a similar name (and find our indexed site) and think they've somehow found actual data (the data in the sanbox isn't real - but it sometimes looks real).